### ISSUE TITLE
regression with 9.2.0-alpha.7

### ISSUE THREAD

**Issue by cexbrayat** [intlify/vue-i18n-next#591]
### Reporting a bug?

The minimum setup (see https://github.com/cexbrayat/vue-v313/pull/4 for a reproduction) displays `app.hello` instead of `Hello`, with a warning:

    Not found 'app.hello' key in 'en' locale messages.

Downgrade to alpha.6, run again, it displays Hello correctly.

### Expected behavior

It should display the provided translation

### Reproduction

https://github.com/cexbrayat/vue-v313/pull/4

### System Info

```shell
System:
    OS: macOS 11.4
    CPU: (8) x64 Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz
    Memory: 394.07 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.17.0 - ~/.volta/tools/image/node/14.17.0/bin/node
    Yarn: 1.22.10 - ~/.volta/tools/image/yarn/1.22.10/bin/yarn
    npm: 6.14.13 - ~/.volta/tools/image/npm/6.14.13/bin/npm
    Watchman: 2021.06.07.00 - /usr/local/bin/watchman
  Browsers:
    Chrome: 92.0.4515.107
    Firefox: 89.0.2
    Safari: 14.1.1
  npmPackages:
    vue: 3.1.5 => 3.1.5
    vue-i18n: 9.2.0-alpha.7 => 9.2.0-alpha.7
```

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/intlify/vue-i18n-next/blob/master/.github/CONTRIBUTING.md)
- [X] Read the [Documentation](https://vue-i18n.intlify.dev/)
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/intlify/vue-i18n-next/discussions)

**Comment by kazupon** [comment]
Thank you for your report!
I've fixed it at `9.2.0-alpha.9`
Please check it!

**Comment by cexbrayat** [comment]
Works great thanks!
